pandas/io/tests/test_sql.py

```diff
@@ -1956,14 +1977,6 @@ def test_datetime_date(self):
         elif self.flavor == 'mysql':
             tm.assert_frame_equal(res, df)
 
-    def test_datetime_time(self):
-        # test support for datetime.time
-        df = DataFrame([time(9, 0, 0), time(9, 1, 30)], columns=["a"])
-        # test it raises an error and not fails silently (GH8341)
-        if self.flavor == 'sqlite':
-            self.assertRaises(sqlite3.InterfaceError, sql.to_sql, df,
-                              'test_time', self.conn)
-
```

Comment:
Can you leave this test here? (but the updated version of course with checking of the output)